R/pairwise-comparisons.R

```diff
@@ -188,7 +219,9 @@ pairwise_comparison_one_group <- function(scores,
                                           by,
                                           ...) {
   if (!("model" %in% names(scores))) {
-    stop("pairwise compairons require a column called 'model'")
+    cli_abort(
+      "pairwise compairons require a column called 'model'"
```

Comment:
```suggestion
      "!" = "pairwise compairons require a column called 'model'"
```

There are a few `cli` statements which are missing a `"!" =`
I'll make some suggested edits when I see them, but otherwise we could also just live that to a future PR

Reply:
Thanks. `cli_abort()` adds it by default, so where we ain't providing extra information, I take advantage of that to avoid the trouble of having to concatenate the bullet (in other words, I'm lazy).